### PR TITLE
feat(col): Allow margin prop

### PR DESCRIPTION
**Pull request by obartra**
margin prop was not allowed in Col because we were setting margin direction
properties and both cannot be set at the same time.

- Not set margin defaults when `margin` prop is passed
- Replace thrown errors with `console.error` to prevent the page from breaking

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -28,6 +28,10 @@ export function getDisplayName(WrappedComponent: string | Component): string {
 }
 
 export function validateSpacingProps(props: SpacingProps) {
+  if (isProd) {
```
Comment (arahansen) [inline src/utils.js]: any idea if this is enough for uglify to dead code eliminate the rest of this function?

Comment (obartra) [inline src/utils.js]: yeah this is removed out of `reflex.min.js` but not from `reflex.js` (since `reflex.js` is not uglified)

Comment (arahansen) [inline src/utils.js]: awesome!

Comment (obartra) [inline src/utils.js]: There are some additional optimizations we could do for reflex but when I tried it we were getting 0.05kb more so I don't think it's worth it

**Pull request merged**